android/src/main/java/com/guichaguri/trackplayer/model/TrackType.kt

```diff
@@ -1,26 +0,0 @@
-package com.guichaguri.trackplayer.model
```

Comment:
There was no reason to have this duplicated class when we can use the library one directly.